### PR TITLE
refactor Player to PlayerModel

### PR DESCRIPTION
**Pull request by CalvinNhieu**

### CODE REVIEW

File: bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/player/PlayerModel.java [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+ * File: PlayerModel.java
+ * Class: PlayerModel
  *
  * Version: 0.0.1
- * Date: September 7th, 2016
+ * Date: September 8th, 2016
  */
 
-package nl.tudelft.bust_a_move20162017.bust_a_move_framework.player;
+package nl.tudelft.bust_a_move20162017.bust_a_move_framework.playermodel;
```
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/player/PlayerModel.java]: Keep player MVC in the same package

Comment (CalvinNhieu) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/player/PlayerModel.java]: What do you mean by player MVC?

**Comment by peace-4-you** [comment]
playerModel and playerView should be in the same package. 
--> package nl.tudelft.bust_a_move20162017.bust_a_move_framework.player;

**Comment by CalvinNhieu** [comment]
I see. There is no PlayerView, but I will revise this in the bubble classes

**Pull request merged**